app/components/concerns/assessment_detailable.rb

```diff
@@ -5,7 +5,7 @@ module AssessmentDetailable
 
   included do
     def assessment_details
-      @assessment_details ||= planning_application.assessment_details_for_review
+      @assessment_details ||= planning_application.assessment_details_for_review.reject { |detail| detail.category == "check_publicity" }
```

Comment:
You can just use `reject(&:check_publicity?)` here as the `enum` creates an instance method for that category